android/CouchbaseLite/src/androidTest/java/com/couchbase/lite/LogTest.java

```diff
@@ -86,31 +125,117 @@ public void run() {
                 }
             }
         });
+    }
+
+    @Test
+    public void testFileLoggingDefaultBinaryFormat() {
+        final File path = new File(
+                context.getCacheDir().getAbsolutePath(),
+                "testFileLoggingDefaultBinaryFormat"
+        );
+        final String logDirectory = emptyDirectory(path.getAbsolutePath());
```

Comment:
i think we should empty the directory at the end of this test.

Reply:
ok

Reply:
no need to empty, since the beginning of the tests are removing the contents, it won't cause any issues. And we are keeping the folder so, we can keep it like this.